src/Tempest/CommandBus/src/functions.php

```diff
@@ -6,6 +6,9 @@
 
     use Tempest\CommandBus\CommandBus;
 
+    /**
+     * Dispatches the given `$command` to the {@see CommandBus}, triggering all associated command handlers.
```

Comment:
Does @see need to be fully qualified?

Reply:
I think so! At least in VSC. But php-cs-fixer isn't happy with that, it simplifies the FQCN. Can you tell me if that works in PHPStorm?

Reply:
I'll double check PHPStorm. CS-fixer is probably something we could change if necessary/there is a comment specific rule.